translations/fr/try_ruby_10.md

```diff
@@ -0,0 +1,26 @@
+---
+lang:   EN
+title:  Vous avez 30 minutes? Essayez Ruby!
```

Comment:
Perhaps we should prefer `tu` wherever we use `vous` in order to use the same everywhere?